wynnset/tapestry-wp
# Fix Import Statment Causing Docker Compliation Failure

As discussed in #575, Linux doesn't like mismatched capitalization. This PR brings the master branch into line with the TYDE branch's capitalization of the `YouTubeMedia.vue` file, so branch switching doesn't break everything.

Steps to test:
- Run `docker-compose up` and verify the `app` container doesn't throw a compile error
